frontend/src/component/feature/FeatureStrategy/FeatureStrategySegment/RecentlyUsedSegments/RecentlyUsedSegments.tsx

```diff
@@ -0,0 +1,68 @@
+import { styled, Typography } from '@mui/material';
+import { useRecentlyUsedSegments } from './useRecentlyUsedSegments';
+import type { ISegment } from 'interfaces/segment';
+import { FeatureStrategySegmentChip } from '../FeatureStrategySegmentChip';
+import { useSegments } from 'hooks/api/getters/useSegments/useSegments';
+import { useUiFlag } from 'hooks/useUiFlag';
+
+type IRecentlyUsedSegmentsProps = {
```

Comment:
Not important, but isn't the `I` prefix only for interfaces? I know we've used it for interfaces, but types are typically without the leading I, I think. I'd suggest either removing the `I` or changing it to an interface. My preference would be removing the prefix, but it's up to you.

```suggestion
type RecentlyUsedSegmentsProps = {
```